doc/epub.md

```diff
@@ -141,8 +141,12 @@ Pandoc has an EPUB3 writer.  It renders LaTeX math into MathML, which
 EPUB3 readers are supposed to support (but unfortunately few do).
 
 Of course, this isn't much help if you want EPUB2 output (`pandoc -t epub2`)
-or target readers that don't support MathML. Then you should try using the
-`--webtex` option, which will use a web service to convert the TeX to an image.
+or target readers that don't support MathML. Then you have two options:
+
+1.  Use the `--webtex` option, which will use a web service to convert the TeX to an image.
+2.  Use the `--gladtex` option to convert maths into SVG images on your local
+    machine. The added benefit is the increased accessibility for blind users
+    that are able to see the LaTeX sources of the formulas instead of the image.
```

Comment:
Is this "added benefit" specific to `--gladtex` or does it apply to `--webtex` too?
(`--webtex` gives you an alt attribute with the original tex.)

Also, as already noted, a link to the part of the manual that gives details on these options would be great.  (Or you could just make `--gladtex` and `--webtex` into links.)